### DISCUSSION TITLE
Previously working setup now broken, no log entries about radio communicatuon

### DISCUSSION THREAD

**Discussion by scorpi11** [tbnobody/OpenDTU#2166] [General]
[edit: solved, I misconfigured the serial number in the inverter settings]

Hi, I use a device with NRF24L01+ radio to talk to my HM600. This worked in the past. Now I flashed the device with the current git version using MS Visual Studio Code and tried to restore the setup. I uploaded a pin profile and the system info page looks fine:

nRF24 Status	configured
nRF24 Chip Status	connected
nRF24 Chip Type	nRF24L01+

I configured WiFi and the admin password. NTP works fine. The green LED on the inverter is blinking.

Now it gets confusing. I added an inverter in "Settings -> Inverter Settings" using the serial number I used in the past.

Then I noticed that there is an additional DTU setting, where a serial number is requested. What is this for? I entered the serial number of my inverter. Is this correct? I don't remember if the old firmware also had separate DTU and inverter settings.

Now I don't see any data from my inverter. The live data page is empty. I also don't see any log output regarding the radio communication on the serial or web console. Any hints what I did wrong?

**Comment by stefan123t** [comment]
@scorpi11 no don’t do that. The DTU ID is used to send the commands from your OpenDTU to the inverter. You maybuse something like 1999 A29 12345 with A29 12345 being the details used in communication with your inverter. I dunno if A for 2024 is already allowed there and 29 is the calendar week of production. 12345 is the serial number of your DTU during that week of production. I believe I have something like 199980012345 but you can choose your own just not the same serial as your inverter ;)

**Reply by scorpi11** [reply]
Thanks, that is how I understood it, too, after going into the details.
